src/main/scala/coreplex/Coreplex.scala

```diff
@@ -67,11 +70,13 @@ abstract class Coreplex(implicit val p: Parameters, implicit val c: CoreplexConf
   val io = new CoreplexIO
 }
 
-class DefaultCoreplex(tp: Parameters, tc: CoreplexConfig) extends Coreplex()(tp, tc) {
+class DefaultCoreplex(tp: Parameters, tc: CoreplexConfig,
+    clockSignal: Clock = null, resetSignal: Bool = null)
+    extends Coreplex(clockSignal, resetSignal)(tp, tc) {
```

Comment:
Maybe shortening _clock and _reset could also let all the tp, tc, _clock, _reset arguments be on the same line?